### PR TITLE
Fix #20 Get Media Likes

### LINKED ISSUE #20: Get media likes

### PR DESCRIPTION
**Pull request by eversionsystems**
Saw the comment suggesting using this endpoint which allows me to get the Instagram username.

/graphql/query/?query_id=17864450716183058&variables={"shortcode":"{{shortcode}}","first":{{count}},"after":"{{likeId}}"}

I've added in a new functions and model "like"

### CODE REVIEW

File: src/InstagramScraper/Instagram.php [hunk 1]
```diff
@@ -387,6 +389,63 @@ public function getMediaCommentsByCode($code, $count = 10, $maxId = null)
...
+			
+            $hasPrevious = $jsonResponse['data']['shortcode_media']['edge_liked_by']['page_info']['has_next_page'];
+            $numberOfLikes = $jsonResponse['data']['shortcode_media']['edge_liked_by']['count'];
+            if ($count > $numberOfLikes) {
+                $count = $numberOfLikes;
+            }
+            if (sizeof($nodes) == 0) {
+                return $likes;
+            }
+            $maxId = $nodes[sizeof($nodes) - 1]['node']['id'];
```
Comment (adgray) [inline src/InstagramScraper/Instagram.php]: This doesn't work for likes (it won't fetch a second batch if $count is more than MAX_LIKES_PER_REQUEST). Instead, use

$maxId = $jsonResponse['data']['shortcode_media']['edge_liked_by']['page_info']['end_cursor'];

**Comment by raiym** [comment]
@eversionsystems thank you for contribution.

Could you please add test that will check if number of likes? 
For example take this media: https://www.instagram.com/p/BblA2Ihg0if
And add test that will pull 500 likes

Thank you

File: src/InstagramScraper/Instagram.php (hunk 1 above)
Comment (eversionsystems) [reply src/InstagramScraper/Instagram.php]: Spot on that fixed it thanks!

Comment (eversionsystems) [reply src/InstagramScraper/Instagram.php]: One question, I set the max likes per batch to 300 to equal the same as comments but what would be a better max batch number to use?

Comment (raiym) [reply src/InstagramScraper/Instagram.php]: @eversionsystems for now it is okay in my opinion. After release we will get feedback if something is wrong.

**Comment by raiym** [comment]
Dear @eversionsystems do you have plans to implements the test or if you don't I can merge and implement them myself

**Comment by eversionsystems** [comment]
@raiym I did a quick test with that image shortcode you suggested and it returned over 6000 usernames for the likes.

**Comment by raiym** [comment]
Okay, cool. 
Then I will merge and implement test.

**Comment by raiym** [comment]
@eversionsystems thank you for contribution

**Pull request merged**